apps/fz_http/test/fz_http_web/live/notifications_live/badge_test.exs

```diff
@@ -0,0 +1,81 @@
+defmodule FzHttpWeb.NotificationsLive.BadgeTest do
+  @moduledoc """
+  Test notifications badge.
+  """
+  use FzHttpWeb.ConnCase, async: true
+  alias FzHttp.Notifications
+
+  setup do
+    on_exit(fn -> Notifications.clear() end)
+  end
+
+  setup [:create_notifications]
+
+  test "badge has no notifications", %{admin_conn: conn} do
+    {:ok, view, _html} = live_isolated(conn, FzHttpWeb.NotificationsLive.Badge)
+
+    html =
+      view
+      |> render()
+
+    assert html =~
+             "<span class=\"icon has-text-grey-dark\"><i class=\"mdi mdi-circle-outline\"></i></span>"
+  end
+
+  test "badge has 5 notifications after adding 5", %{
+    admin_conn: conn,
+    notifications: notifications
+  } do
+    {:ok, view, _html} = live_isolated(conn, FzHttpWeb.NotificationsLive.Badge)
+
+    for notification <- notifications do
+      Notifications.add(notification)
+    end
+
+    html =
+      view
+      |> render()
```

Comment:
```suggestion
    {:ok, _view, html} = live_isolated(conn, FzHttpWeb.NotificationsLive.Badge)

    for notification <- notifications do
      Notifications.add(notification)
    end
```

Reply:
I believe you have to re-render the html to get the updated markup from the view?

Reply:
Yup, my brain stopped working after seeing the first one.

Reply:
However since we aren't asserting anything after first render, we might as well just move the first render below the add.